rllib/tests/test_io.py

```diff
@@ -80,20 +80,6 @@ def test_agent_output_logdir(self):
             agent = self.write_outputs("logdir", fw)
             self.assertEqual(len(glob.glob(agent.logdir + "/output-*.json")), 1)
 
-    def test_agent_output_infos(self):
```

Comment:
as it turns out there was a way to store infos in tensorflow for training.
Its not publicly documented, but there is 1 test for it, so there were
2 occurences in the codebase. This whole pr pretty much disables you from being able
to write env infos out in a sample batch, so we'll allow it to still be used for training,
but not allow it for output writing.

I removed the test where we check for env infos in the output batch for the time being.

cc @gjoliver